src/services/professorInfo.ts

```diff
@@ -0,0 +1,41 @@
+import { PrismaClient, ProfessorInfo } from '.prisma/client';
+import {
+  CreateProfessorInfoDto,
+  UpdateProfessorInfoDto,
+} from '@/models/professorInfo.model';
+
+const prisma = new PrismaClient();
+
+export const getProfessInfoForUser = async (
+  userId: number,
+): Promise<ProfessorInfo | 'not found'> => {
+  const info = await prisma.professorInfo.findUnique({ where: { userId } });
+  return info || 'not found';
+};
+
+export const createProfessorInfo = async (
+  info: CreateProfessorInfoDto,
+): Promise<ProfessorInfo | 'already exists'> => {
```

Comment:
is it dangerous to assume the cases will always be these two options though(success or already exists)?